client/src/components/FlagshipMenu/Flagship.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Menu from './Flagship';
+
+
+storiesOf('Flagship Menu', module)
+    .add('Primary', () => (<Menu city='Baltimore, MD'
+    							 image=''
+    							 address='1429 Aliceanna Street Baltimore, MD 21231' 
+    							 phone='(443) 447-8929' 
+    							 email='baltimore@moderntrousseau.com' 
+    							 instagram='Need more inspo? Follow us at @moderntrousseaubaltimore'
```

Comment:
I think the text here should be built-in, and maybe just the handle should be passed down as a prop

Reply:
Also, if you do not already have it please install [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) for formatting.

Reply:
I'm not sure I understand what you mean @ZakkMan . Do you want me to move the text to the component file?

Reply:
Someone should just pass in `instagram='@moderntrousseaubaltimore'` and then have that construct the whole string inside the component (`'Need more inspo? Follow us at @moderntrousseaubaltimore'`)